hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -236,6 +238,17 @@ public Time getTime(String columnLabel) {
 	@Override
 	public Timestamp getTimestamp(String columnLabel) {
 		LocalDateTime localDateTime = row.getLocalDateTime(columnLabel);
+		if (localDateTime==null) {
+			try {
+				OffsetDateTime offsetDateTime = row.getOffsetDateTime(columnLabel);
+				if (offsetDateTime!=null) {
+					localDateTime = LocalDateTime.from(offsetDateTime);
+				}
+			}
+			catch (Exception e) {
+				//MySQL client throws a nasty NPE here
```

Comment:
@gavinking  Can't we do anything better than catch a NullPointer exception?

Reply:
We don't want to. This is what we want to happen.

Reply:
I'm pretty sure that this is not how I usually want to deal with NullPointerException, thank you very much.
If you could explain to me why we have to deal with it this way, it would be nice.

Reply:
Because (1) we're just trying it as a fallback, and (2) the NPE is a bug in the Vert.x MySQL client.

I'm not actually a beginner at this.

Reply:
And FYI it's `Exception` instead of `NullPointerException` because Vert.x is actually trying and failing to throw an `UnsupportedOperationException` but they might fix that.